api/scpca_portal/templates/readme.md

```diff
@@ -34,9 +34,28 @@ For instructions on using the RDS files, please see [FAQ: How do I use the provi
 
 If you identify issues with this download, please [file an issue on GitHub.](https://github.com/AlexsLemonade/scpca-portal/issues/new) If you would prefer to report issues via e-mail, please contact us at [scpca@ccdatalab.org](mailto:scpca@ccdatalab.org).
 
-## Citing
+## Citation
+
+You must cite the data submitter using language provided as part of the project abstract (as applicable), the publication listed for the project (as applicable), or both, and the ScPCA Portal using the language below.
+For more information, please see [the How to Cite section of our documentation](https://scpca.readthedocs.io/en/latest/citation.html).
```

Comment:
This would point to whatever is in the `main` branch of `scpca-docs`, not whatever the default version of the docs is at the time. I suspect this is just fine for this particular case, but I'm also not aware of a good alternative strategy.

Reply:
I would think that we would want to point to the latest version of the docs though?

We could take this as a template variable as well.  Would we need to map different projects / samples to different documentation versions for any reason?

Reply:
The `latest` version of docs will be what's in `main`, not the most recent tag, which will be the default version of the docs (see https://github.com/AlexsLemonade/scpca-portal/issues/468).   (That should _nearly always_ be _functionally the same_.) I don't see a case where we would map different projects or samples to different versions of the docs.

Reply:
@davidsmejia I'll see you in `#scpca-portal` in Slack; I think I figured something out in RTD...

Reply:
I'm going to change this to:

```suggestion
For more information, please see [the How to Cite section of our documentation](https://scpca.readthedocs.io/en/stable/citation.html).
```

Because, per David's suggestion, we will version with tags formatted `YYYY.MM.DD`, which is semantic versioning of dates.